modules/using-cli.adoc

```diff
@@ -0,0 +1,221 @@
+
+[discrete]
+=== Backing up Central database
+
+Run the following command to backup Central's database:
+[source,terminal]
+----
+$ $ roxctl -e "$ROX_CENTRAL_ADDRESS" central backup
+----
+
+[discrete]
+=== Restoring Central database
+
+Run the following command to restore Central's database:
+[source,terminal]
+----
+$ $ roxctl -e "$ROX_CENTRAL_ADDRESS" central db restore <backup-filename>
+----
+
+[id="manage-secure-clusters_{context}"]
+== Managing secured clusters
+
+To secure a Kubernetes or an {ocp} cluster, you must deploy {product-title} services into the cluster.
+You can generate deployment files in the RHACS portal by navigating to the *Platform Configuration* -> *Clusters* view, or you can use the `roxctl` CLI.
+
+[discrete]
+=== Generating sensor deployment YAML file
+
+.Kubernetes
+
+[source,terminal]
+----
+$ $ roxctl -e "$ROX_CENTRAL_ADDRESS" sensor generate k8s --name <cluster-name> --central "$ROX_CENTRAL_ADDRESS"
+----
+
+.{ocp}
+
+[source,terminal]
+----
+$ $ roxctl -e "$ROX_CENTRAL_ADDRESS" sensor generate openshift --name <cluster-name> --central "$ROX_CENTRAL_ADDRESS"
+----
+
+Read the `--help` output to see other options that you may need to use depending on your system architecture.
+
+Make sure that the endpoint you provide for `--central` can be reached from the cluster where you are deploying {product-title} services.
+
+[NOTE]
+====
+If you are using a non-gRPC capable LoadBalancer, such as HAProxy, AWS Application Load Balancer (ALB), or AWS Elastic Load Balancing (ELB):
+
+* Use the WebSocket Secure (`wss`) protocol.
+To use `wss`, prefix the address with *`wss://`*, and
+* Add the port number after the address, for example,
++
+[source,terminal]
+----
+$ roxctl sensor generate k8s --central wss://stackrox-central.example.com:443
+----
+====
+
+[discrete]
+=== Downloading sensor bundle for existing clusters
+
+Use the following command to download sensor bundles for existing clusters by specifying a cluster name or ID.
+
+[source,terminal]
+----
+$ roxctl sensor get-bundle <cluster-name-or-id>
+----
+
+[discrete]
+=== Deleting cluster integration
+
+[source,terminal]
+----
+$ $ roxctl -e "$ROX_CENTRAL_ADDRESS" cluster delete --name=<cluster-name>
+----
+
+[IMPORTANT]
+====
+Deleting cluster integration will not remove {product-title} services running in the cluster.
+You can remove them by running the `delete-sensor.sh` script from the sensor installation bundle.
+====
+
+[id="check-policy-compliance_{context}"]
+== Checking policy compliance
+
+[discrete]
+=== Checking deployment YAML files
+
+The following command checks build-time and deploy-time violations of your security policies in YAML deployment files.
+//TODO: Add link to security policies section
+Use this command to validate:
+
+* Configuration options in a YAML file, such as resource limits or privilege options;
+or
+* Aspects of the images used in a YAML file, such as components or vulnerabilities.
++
```

Comment:
Is this command block supposed to be paired with the above bullet? If not, you'll want to get rid of this `+`